internal/commands/app/create_inputs.go

```diff
@@ -30,23 +32,34 @@ var (
 
 type createInputs struct {
 	newAppInputs
-	Directory  string
-	DataSource string
-	DryRun     bool
+	Directory string
+	Cluster   string
+	DataLake  string
+	DryRun    bool
 }
 
-type dataSource struct {
-	Name   string           `json:"name"`
-	Type   string           `json:"type"`
-	Config dataSourceConfig `json:"config"`
+type clusterService struct {
```

Comment:
Yeah, I went with the postfix `Service` (since that is what the internal team knows this as) and to add as a secondary differentiator this isn't a current pkg version of a `Cluster` or `DataLake`.

Reply:
so obviously "service" is a bit of an overloaded term in our ecosystem ... I still tend to think of anything with that name as something that _does something_, rather than _is something_ (if that makes sense...)

again, feel free to leave as-is.  i see in the not-so-distant future this potentially moving into `local` package alongside the other app resources, but I don't want to go crazy with that stuff just yet

Reply:
I am going to go with the offline suggestions of dataSource* for clarity even though it is on the verbose side.